pkg/client/entitlement.go

```diff
@@ -51,6 +52,7 @@ func (c *client) SearchEntitlements(ctx context.Context, filter *SearchEntitleme
 		PageToken:        nil,
 		Query:            stringPtr(filter.Query),
 		AppDisplayName:   stringPtr(filter.AppDisplayName),
+		IncludeDeleted:   &filter.IncludeDeleted,
```

Comment:
I'm very confused by this &. Isn't this a bool prop?

Reply:
Yeah but RequestCatalogSearchServiceSearchEntitlementsRequest wants pointers to everything.